apps/users/migrations/0002_profile.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.0.3 on 2020-02-21 02:45
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0001_replace_django_user_table'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Profile',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('bio', models.CharField(max_length=620)),
```

Comment:
Should we make this a Text field instead of a CharField so that people may have long bios?